src/spikeinterface/sortingcomponents/tests/common.py

```diff
@@ -0,0 +1,22 @@
+from spikeinterface.core import generate_ground_truth_recording
+
+
+def make_dataset():
+    # this replace the MEArec 10s file for testing
+    recording, sorting = generate_ground_truth_recording(
+        durations=[30.0],
```

Comment:
Could we make this shorter?
I was hoping this would be faster than the Meraec thing as this module together with core sorters is the slowetst one in the tests.